tutorials/notebooks/specutils-fit-line/specutils-fit-line1.ipynb

```diff
@@ -0,0 +1,475 @@
+{
```

Comment:
<p>What's the motivation for calling it <code>xid</code>.  Is there a more descriptive variable name we could use? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astropy/astropy-tutorials/pull/491/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>How about <code>crossmatch_id</code> ?</p>